romancal/stpipe/core.py

```diff
@@ -7,6 +7,9 @@ class RomanStep(Step):
     """
     Base class for Roman calibration pipeline steps.
     """
+    spec = """
+    output_ext =  string(default='.asdf')
```

Comment:
The trailing comment is actually important -- it's used as the description of the parameter in the strun help output.  I don't think it's inherited from Step now that we've overwritten output_ext, so we should probably keep it here as well.